### ISSUE TITLE
Customize `warnBadFormat` message

### ISSUE THREAD

**Issue by julioz** [scalameta/sbt-scalafmt#80]
Thanks for `scalafmt`, it's been pretty useful for us!

Following the workflow of one of our teams, we have some specific commands we would like engineers to run before/while formatting their scala codebases; it's been a recurring question what commands are these, and we thought it would be great to [be able to configure the message](https://github.com/scalameta/sbt-scalafmt/blob/master/plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala#L218) `scalafmt` outputs when bad formatting is detected.

So, in summary, could we have a plugin option in which we pass a custom message to change
```
[warn] /data/git-repositories/<path_to_file>.scala isn't formatted properly!
```
into
```
[warn] /data/git-repositories/<path_to_file>.scala isn't formatted properly! Run <some_command> to auto-format it.
```

**Comment by poslegm** [comment]
Hmm... It is easy to implement, but is it really necessary? I think that formatting command in the project README is more simple way to achieve learn your teammates than custom messages in logs.

**Comment by julioz** [comment]
@poslegm we use it in several projects each with different setups and teams (which makes it 'teaching' a bit more cumbersome). The type of command I have in mind are `make`/`sbt` targets and custom internal tool parameters.

**Comment by poslegm** [comment]
Ok, I understand you. 

Implementation TODO list:
1. Add new `taskKey` ([example](https://github.com/scalameta/sbt-scalafmt/blob/master/plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala#L36))
2. Concatenate `taskKey` value with [log message](https://github.com/scalameta/sbt-scalafmt/blob/master/plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala#L218)
3. Add `taskKey` description into [docs](https://scalameta.org/scalafmt/docs/installation.html#task-keys)

Fell free to contribute!
